spec/system/consumer/split_checkout_tax_incl_spec.rb

```diff
@@ -139,11 +139,65 @@
           end
 
           # DB check
-          order_within_zone.reload
-          voucher_adjustment = order_within_zone.voucher_adjustments.first
+          assert_db_voucher_adjustment(-10.00, -1.15)
+        end
+
+        describe "moving between summary to summary via edit cart" do
+          let!(:voucher) do
+            create(:voucher, code: 'good_code', enterprise: distributor, amount: 15)
+          end
+
+          it "recalculate the tax component properly" do
+            visit checkout_step_path(:details)
+            proceed_to_payment
+
+            # add Voucher
+            fill_in "Enter voucher code", with: voucher.code
+            click_button("Apply")
+
+            proceed_to_summary
+
+            assert_db_voucher_adjustment(-10.00, -1.15)
+
+            # Click on edit link
+            within "div", text: /Order details/ do
+              # It's a bit brittle, but the scoping doesn't seem to work
+              all(".summary-edit").last.click
+            end
+
+            # Update quantity
+            within ".cart-item-quantity" do
+              input = find(".line_item_quantity")
+              input.send_keys :up
```

Comment:
I wonder the method used here works better? 

https://github.com/openfoodfoundation/openfoodnetwork/blob/master/spec/system/consumer/shopping/cart_spec.rb#L204-L206

Reply:
I changed it to the other method as it's more explicit, but in the end I think it's probably same same.